api/src/pcapi/core/fraud/factories.py

```diff
@@ -166,6 +167,19 @@ class Meta:
         else users_models.EligibilityType.AGE18
     )
 
+    @classmethod
+    def generate_content_from_user(cls, factory_class: typing.Type[factory.Factory], user: users_models.User) -> dict:
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Cool l'injection de dépendance